pylabrobot/liquid_handling/backends/hamilton/STAR_backend.py

```diff
@@ -7285,6 +7285,186 @@ def mm_to_z_drive_increment(value_mm: float) -> int:
   def z_drive_increment_to_mm(value_increments: int) -> float:
     return round(value_increments * STARBackend.z_drive_mm_per_increment, 2)
 
+  async def clld_probe_y_position_using_channel(
+    self,
+    channel_idx: int,  # 0-based indexing of channels!
+    probing_direction: Literal["forward", "backward"],
+    start_pos_search: Optional[float] = None,  # mm
+    end_pos_search: Optional[float] = None,  # mm
+    channel_speed: float = 10.0,  # mm/sec
+    channel_acceleration_int: Literal[1, 2, 3, 4] = 4,  # * 5_000 steps/sec**2 == 926 mm/sec**2
+    detection_edge: int = 10,
+    current_limit_int: Literal[1, 2, 3, 4, 5, 6, 7] = 7,
+    post_detection_dist: float = 2.0,  # mm,
+    tip_bottom_diameter: float = 1.2,  # mm
+  ) -> float:
+    """
+    Probes the y-position at which a conductive material is detected using the channel's capacitive
+    Liquid Level Detection (cLLD) capability.
+
+    This method aims to provide safe probing within defined boundaries to avoid collisions or damage
+    to the system. It is specifically designed for conductive materials.
+
```

Comment:
```suggestion

    This method aims to provide safe probing within defined boundaries to avoid collisions or damage
    to the system. It is specifically designed for conductive materials.
    
    This method assumes you have mounted the teaching needles on the specified channel. If you have a different tip, you might need to change the `tip_bottom_diameter` parameter.

```